cloud/azure/modules/app/variables.tf

```diff
@@ -76,17 +65,18 @@ variable "resource_group_name" {
 variable "postgres_admin_login" {
   type        = string
   description = "Postgres admin login"
+  default     = "psqladmin"
 }
 
 variable "postgres_sku_name" {
   type        = string
   description = "The sku name for postgres server"
-  default     = "GP_Gen5_2"
+  default     = "GP_Standard_D2s_v3"
 }
 variable "postgres_storage_mb" {
   type        = number
   description = "The mb of storage for postgres instance"
-  default     = 5120
+  default     = 32768
```

Comment:
This is pretty large. After three years in prod Seattle's database is only 275MB out of 5GB available. Our old, pre-terraform install had prod at 50GB. Waste of money. 

Can the storage size be expanded on demand or does it have to be fully allocated upfront when created?

Reply:
This is the minimum allocation possible for postgres_flexible_database. 
<img width="683" alt="Screenshot 2024-08-21 at 9 18 22 AM" src="https://github.com/user-attachments/assets/e326b625-a8ca-48f8-b5c4-54fd78a98ead">

Reply:
Huh. The cloud is crazy. Thanks for the info! 🙂